### PR TITLE
Migrate Docker images to Ubuntu 24.04

### PR DESCRIPTION
**Pull request by rubo**
## Changes

Migrated Docker images to Ubuntu 24.04 (Noble Numbat)

#### What types of changes does your code introduce?

- [ ] Bugfix (a non-breaking change that fixes an issue)
- [ ] New feature (a non-breaking change that adds functionality)
- [ ] Breaking change (a change that causes existing functionality not to work as expected)
- [ ] Optimization
- [ ] Refactoring
- [ ] Documentation update
- [ ] Build-related changes
- [x] Other: Docker images migration

#### Requires testing

- [ ] Yes
- [x] No

#### If yes, did you write tests?

- [ ] Yes
- [x] No

#### Notes on testing

Needs further manual testing.

#### Requires documentation update

- [ ] Yes
- [x] No

#### Requires explanation in Release Notes

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by LukaszRozmej — APPROVED** [review]
Let's also wait for @kamilchodola approval

**Comment by kamilchodola** [comment]
Started one VM with that. Looks like it works on proper version. Node is syncing well for now.

```
root@9319038cfb1e:/nethermind# cat /etc/os-release
PRETTY_NAME="Ubuntu 24.04 LTS"
NAME="Ubuntu"
VERSION_ID="24.04"
VERSION="24.04 LTS (Noble Numbat)"
VERSION_CODENAME=noble
ID=ubuntu
ID_LIKE=debian
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
UBUNTU_CODENAME=noble
LOGO=ubuntu-logo
```

**Comment by rubo** [comment]
Alright. I don't expect any issues. I'll merge this tomorrow, so it will be used by more people and we'll catch issues quciker

**Pull request merged**
